Baby Monitor/Source Files/Modules/Dashboard/DashboardView.swift

```diff
@@ -182,7 +182,7 @@ extension Reactive where Base: DashboardView {
     
     var connectionStatus: Binder<Bool> {
         return Binder(base) { dashboardView, isConnection in
-            isConnection ? dashboardView.updateConnectionStatus(isConnected: true) : dashboardView.updateConnectionStatus(isConnected: false)
+            dashboardView.updateConnectionStatus(isConnected: isConnection)
```

Comment:
`isConnection` sounds a bit weird, maybe `isConnected`? Here and in other places in this PR.

Reply:
👍